bin/numba

```diff
@@ -22,8 +22,8 @@ def make_parser():
                         help='[Deprecated] Dump the control flow graph')
     parser.add_argument('--dump-ast', action="store_true",
                         help='[Deprecated] Dump the AST')
-    parser.add_argument('--fancy', action='store_true',
-                        help='Try to output fancy files (.dot or .html)')
+    parser.add_argument('--fancy', nargs=1, metavar='html_filename',
```

Comment:
Ah, okay. I wasn't able to get it to work before without the nargs=1, but now it does (I think I had action='store_true' before which it definitely doesn't like).